plugins/kubernetes/app/models/kubernetes/role_config_file.rb

```diff
@@ -0,0 +1,173 @@
+module Kubernetes
+
+  # This file represents a Kubernetes configuration file for a specific project role.
+  # A single configuration file can have both a Replication Controller and a Service, as two separate documents.
+  #
+  # For the schema definition of a Replication Controller see:
+  # https://cloud.google.com/container-engine/docs/replicationcontrollers/operations
+  #
+  # For the schema definition of a Service, see:
+  # https://cloud.google.com/container-engine/docs/services/operations
+  #
+  class RoleConfigFile
```

Comment:
Hmm, this class seems to duplicate a chunk of the logic in ReleaseDoc.  Are you thinking of refactoring to combine them, or keep them separate?